test/e2e/continue_on_error_test.go

```diff
@@ -90,6 +98,25 @@ func continueOnErrorTest(_ client.Client, invConfig InventoryConfig, inventoryNa
 			},
 		},
 		// Note: No WaitTask when apply fails
+		// TODO: why no wait after create tho?
```

Comment:
Should this commented out stuff be removed?

Reply:
I'm still not exactly sure why the WaitTask isn't being triggered.
Is it not waiting because it's the last apply in the task list?
Is it not waiting because there's no timeout configured?
Why does it wait sometimes and not others?

I started a chat convo about this last week and the consensus from Config Sync participants seemed to indicate that we should change it to always wait. I think that makes the most sense from a user perspective and is both intuitive and less surprising. But I didn't want to make that change in this commit. So I just left the WaitTasks commented out (as I had in other tests).

Do you want me to rip out the comment and just add it in the next PR that enables waiting?